__tests__/api.test.js

```diff
@@ -231,9 +231,53 @@ describe("GET api/reviews where filter matches query", () => {
         expect(output).toEqual(filteredOutput);
       });
   });
+  it("should return an array of objects only containg matched query column value -- owner", () => {
+    return request(app)
+      .get("/api/reviews?owner=mallionaire")
+      .expect(200)
+      .then(({ body }) => {
+        const output = body.reviews;
+        const filteredOutput = output.filter(
+          review => review.owner === "mallionaire"
+        );
+        expect(output).toEqual(filteredOutput);
+      });
+  });
+  it("should return an array of objects only containg matched query column value -- votes", () => {
+    return request(app)
+      .get("/api/reviews?votes=5")
+      .expect(200)
+      .then(({ body }) => {
+        const output = body.reviews;
+        const filteredOutput = output.filter(review => review.votes === 5);
+        expect(output).toEqual(filteredOutput);
+      });
+  });
+  it("should return an array of objects only containg the last matched query column value", () => {
+    return request(app)
+      .get("/api/reviews?votes=5&category=social+deduction&owner=mallionaire")
+      .expect(200)
+      .then(({ body }) => {
+        const output = body.reviews;
+        expect(output[0]).toEqual(
+          expect.objectContaining({
+            review_id: expect.any(Number),
+            title: expect.any(String),
+            category: "social deduction",
+            designer: expect.any(String),
+            owner: "mallionaire",
+            review_img_url: expect.any(String),
+            created_at: expect.any(String),
+            votes: 5,
+            comment_count: expect.any(String),
```

Comment:
if we are expecting every item in output to have these values we should be using something like .forEach() or .every().  also then we need to check array length >0